### PR TITLE
Use marketplace-constants as the source of carrier mcc/mnc/spn (bug 1162499)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1162499

### CODE REVIEW

**Comment by mstriemer** [comment]
close/reopen to trigger Travis.

File: src/media/js/mobilenetwork.js [hunk 1]
```diff
@@ -346,10 +38,6 @@ define('mobilenetwork',
             }
         }
 
-        // Make them integers.
-        mcc = +mcc || 0;
```
Comment (mstriemer) [inline src/media/js/mobilenetwork.js]: We don't need them as integers anymore?

**Comment by mstriemer** [comment]
r+

File: src/media/js/mobilenetwork.js (hunk 1 above)
Comment (diox) [inline src/media/js/mobilenetwork.js]: No, as long as it matches the keys in `carrierRegions` we don't care. When moving them to `marketplace-constants` I just left the default behaviour, which is to format the objects as json, and they ended up being strings, so I removed the conversion to integers here.

Comment (mstriemer) [inline src/media/js/mobilenetwork.js]: :+1: Much better this way.

**Pull request merged**